store/tikv/2pc.go

```diff
@@ -771,6 +805,20 @@ func (c *twoPhaseCommitter) execute(ctx context.Context) error {
 	return nil
 }
 
+func (c *twoPhaseCommitter) doActionWithRetry(ctx context.Context, f func(ctx context.Context) error) (err error) {
+	var retryCnt uint64
+	for {
+		err = f(ctx)
+		if retryCnt < atomic.LoadUint64(&config.GetGlobalConfig().Max2PCRetry) &&
+			err != nil &&
+			strings.Contains(err.Error(), txnRetryableMark) {
+			retryCnt++
+			continue
+		}
+		return
```

Comment:
Could we add some comments to describe when would we meet `txnRetryableMark`?
And too many `&&` make me very tired to remember the conditions before, How about implement it as following
```
func (c *twoPhaseCommitter) doActionWithRetry(ctx context.Context, doAction func(ctx context.Context) error) (err error) {
	max2PcRetry := atomic.LoadUint64(&config.GetGlobalConfig().Max2PCRetry);
	for retryCnt := 0; retryCnt < max2PcRetry; retry++ {
		err = doAction(ctx)
		if err == nil || !strings.Contains(err.Error(),txnRetryableMark){
			return 
		}
	}
}
```

Reply:
`Max2PCRetry` could not be set to 0, if we implement it like this. The action needs to do at least once. Or we change the name to `Max2PCAttempts`, then the range of it could be 1 to max uint64. But I think the current one is better.